lib/floe/awesome_runner.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require "concurrent/array"
+
+module Floe
+  class AwesomeProcess < Thread
```

Comment:
would have preferred not using a thread per process and not using an `AwesomeSpan` centric model.